dropshot/tests/test_demo.rs

```diff
@@ -727,6 +730,57 @@ async fn test_untyped_body() {
     testctx.teardown().await;
 }
 
+// Test `StreamingBody`.
+#[tokio::test]
+async fn test_streaming_body() {
+    let api = demo_api();
+    let testctx = common::test_setup("test_streaming_body", api);
+    let client = &testctx.client_testctx;
+
+    // Success case: empty body
+    let mut response = client
+        .make_request_with_body(
+            Method::PUT,
+            "/testing/streaming_body",
+            "".into(),
+            StatusCode::OK,
+        )
+        .await
+        .unwrap();
+    let json: DemoStreaming = read_json(&mut response).await;
+    assert_eq!(json.nbytes, 0);
+
+    // Success case: non-empty content
+    let body = vec![0u8; 1024];
+    let mut response = client
+        .make_request_with_body(
+            Method::PUT,
+            "/testing/streaming_body",
+            body.into(),
+            StatusCode::OK,
+        )
+        .await
+        .unwrap();
+    let json: DemoStreaming = read_json(&mut response).await;
+    assert_eq!(json.nbytes, 1024);
+
+    // Error case: body too large.
+    let big_body = vec![0u8; 1025];
+    let error = client
+        .make_request_with_body(
+            Method::PUT,
+            "/testing/untyped_body",
```

Comment:
Was this supposed to be `streaming_body`?

Reply:
Whoops yeah! Fixed.